defaults/main.yml

```diff
@@ -1,5 +1,9 @@
 # SPDX-License-Identifier: MIT
 ---
+
+# Linux Pacemaker shell, pcs or crmsh
+ha_cluster_pacemaker_shell: pcs
```

Comment:
I would drop 'pacemaker' here, as pcs handles also corosync, sbd and other cluster components. Crmsh should be able to configure other components than pacemaker, too, I think.

Reply:
@tomjelinek I was following nomenclature from elsewhere in Linux Pacemaker projects. I don't mind altering this var name, but from an end user perspective they would refer to 'use Linux Pacemaker' because that is the owning name of all subprojects, sideprojects and components.

For example....

`"pcs and crm shell are two popular higher-level command-line interfaces to Pacemaker."`
https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Administration/html/pcs-crmsh.html

`"crmsh is a cluster management shell for the Pacemaker High Availability stack."`
https://crmsh.github.io/

Reply:
Also, I can add the identifier to all new (mostly placeholder) files.

Reply:
@sean-freeman That makes sense. Let's keep it the `ha_cluster_pacemaker_shell` then.